### DISCUSSION TITLE
W0910 21:18:56.955332 3317990 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.0, Runtime API Version: 10.2 W0910 21:18:56.955994 3317990 gpu_resources.cc:91] device: 0, cuDNN Version: 8.3.

### DISCUSSION THREAD

**Discussion by juneana123** [PaddlePaddle/PaddleOCR#12674] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu20.04
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：PaddleOCR-release-2.6
- 运行指令/Command Code：
- python tools/train.py -c configs/rec/rec_svtrnet_ch.yml
- 完整报错/Complete Error Message：
[2023/09/10 21:22:08] ppocr INFO: Architecture : 
[2023/09/10 21:22:08] ppocr INFO:     Backbone : 
[2023/09/10 21:22:08] ppocr INFO:         depth : [3, 6, 3]
[2023/09/10 21:22:08] ppocr INFO:         embed_dim : [64, 128, 256]
[2023/09/10 21:22:08] ppocr INFO:         img_size : [32, 320]
[2023/09/10 21:22:08] ppocr INFO:         last_stage : True
[2023/09/10 21:22:08] ppocr INFO:         local_mixer : [[7, 11], [7, 11], [7, 11]]
[2023/09/10 21:22:08] ppocr INFO:         mixer : ['Local', 'Local', 'Local', 'Local', 'Local', 'Local', 'Global', 'Global', 'Global', 'Global', 'Global', 'Global']
[2023/09/10 21:22:08] ppocr INFO:         name : SVTRNet
[2023/09/10 21:22:08] ppocr INFO:         num_heads : [2, 4, 8]
[2023/09/10 21:22:08] ppocr INFO:         out_channels : 96
[2023/09/10 21:22:08] ppocr INFO:         out_char_num : 40
[2023/09/10 21:22:08] ppocr INFO:         patch_merging : Conv
[2023/09/10 21:22:08] ppocr INFO:         prenorm : False
[2023/09/10 21:22:08] ppocr INFO:     Head : 
[2023/09/10 21:22:08] ppocr INFO:         name : CTCHead
[2023/09/10 21:22:08] ppocr INFO:     Neck : 
[2023/09/10 21:22:08] ppocr INFO:         encoder_type : reshape
[2023/09/10 21:22:08] ppocr INFO:         name : SequenceEncoder
[2023/09/10 21:22:08] ppocr INFO:     Transform : None
[2023/09/10 21:22:08] ppocr INFO:     algorithm : SVTR
[2023/09/10 21:22:08] ppocr INFO:     model_type : rec
[2023/09/10 21:22:08] ppocr INFO: Eval : 
[2023/09/10 21:22:08] ppocr INFO:     dataset : 
[2023/09/10 21:22:08] ppocr INFO:         data_dir : /media/wj/newvolume/REC_DATASET/RCTW_rec/RCTW_recval_imgs/
[2023/09/10 21:22:08] ppocr INFO:         label_file_list : ['/media/wj/newvolume/REC_DATASET/RCTW_rec/rec_val.txt']
[2023/09/10 21:22:08] ppocr INFO:         name : SimpleDataSet
[2023/09/10 21:22:08] ppocr INFO:         transforms : 
[2023/09/10 21:22:08] ppocr INFO:             DecodeImage : 
[2023/09/10 21:22:08] ppocr INFO:                 channel_first : False
[2023/09/10 21:22:08] ppocr INFO:                 img_mode : BGR
[2023/09/10 21:22:08] ppocr INFO:             CTCLabelEncode : None
[2023/09/10 21:22:08] ppocr INFO:             SVTRRecResizeImg : 
[2023/09/10 21:22:08] ppocr INFO:                 image_shape : [3, 32, 320]
[2023/09/10 21:22:08] ppocr INFO:                 padding : True
[2023/09/10 21:22:08] ppocr INFO:             KeepKeys : 
[2023/09/10 21:22:08] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2023/09/10 21:22:08] ppocr INFO:     loader : 
[2023/09/10 21:22:08] ppocr INFO:         batch_size_per_card : 256
[2023/09/10 21:22:08] ppocr INFO:         drop_last : False
[2023/09/10 21:22:08] ppocr INFO:         num_workers : 2
[2023/09/10 21:22:08] ppocr INFO:         shuffle : False
[2023/09/10 21:22:08] ppocr INFO: Global : 
[2023/09/10 21:22:08] ppocr INFO:     cal_metric_during_train : True
[2023/09/10 21:22:08] ppocr INFO:     character_dict_path : ppocr/utils/baidu_rec_dict.txt
[2023/09/10 21:22:08] ppocr INFO:     character_type : ch
[2023/09/10 21:22:08] ppocr INFO:     checkpoints : None
[2023/09/10 21:22:08] ppocr INFO:     distributed : False
[2023/09/10 21:22:08] ppocr INFO:     epoch_num : 50
[2023/09/10 21:22:08] ppocr INFO:     eval_batch_step : [0, 2000]
[2023/09/10 21:22:08] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2023/09/10 21:22:08] ppocr INFO:     infer_mode : False
[2023/09/10 21:22:08] ppocr INFO:     log_smooth_window : 20
[2023/09/10 21:22:08] ppocr INFO:     max_text_length : 100
[2023/09/10 21:22:08] ppocr INFO:     pretrained_model : None
[2023/09/10 21:22:08] ppocr INFO:     print_batch_step : 1
[2023/09/10 21:22:08] ppocr INFO:     save_epoch_step : 10
[2023/09/10 21:22:08] ppocr INFO:     save_inference_dir : None
[2023/09/10 21:22:08] ppocr INFO:     save_model_dir : ./output/myrec/rec_svtr_RC
[2023/09/10 21:22:08] ppocr INFO:     save_res_path : ./output/myrec/rec_svtr_RC/predicts_svtr_tiny_ch_all.txt
[2023/09/10 21:22:08] ppocr INFO:     use_gpu : True
[2023/09/10 21:22:08] ppocr INFO:     use_space_char : True
[2023/09/10 21:22:08] ppocr INFO:     use_visualdl : True
[2023/09/10 21:22:08] ppocr INFO: Loss : 
[2023/09/10 21:22:08] ppocr INFO:     name : CTCLoss
[2023/09/10 21:22:08] ppocr INFO: Metric : 
[2023/09/10 21:22:08] ppocr INFO:     main_indicator : acc
[2023/09/10 21:22:08] ppocr INFO:     name : RecMetric
[2023/09/10 21:22:08] ppocr INFO: Optimizer : 
[2023/09/10 21:22:08] ppocr INFO:     beta1 : 0.9
[2023/09/10 21:22:08] ppocr INFO:     beta2 : 0.99
[2023/09/10 21:22:08] ppocr INFO:     epsilon : 8e-08
[2023/09/10 21:22:08] ppocr INFO:     lr : 
[2023/09/10 21:22:08] ppocr INFO:         learning_rate : 0.0005
[2023/09/10 21:22:08] ppocr INFO:         name : Cosine
[2023/09/10 21:22:08] ppocr INFO:         warmup_epoch : 2
[2023/09/10 21:22:08] ppocr INFO:     name : AdamW
[2023/09/10 21:22:08] ppocr INFO:     no_weight_decay_name : norm pos_embed
[2023/09/10 21:22:08] ppocr INFO:     one_dim_param_no_weight_decay : True
[2023/09/10 21:22:08] ppocr INFO:     weight_decay : 0.05
[2023/09/10 21:22:08] ppocr INFO: PostProcess : 
[2023/09/10 21:22:08] ppocr INFO:     name : CTCLabelDecode
[2023/09/10 21:22:08] ppocr INFO: Train : 
[2023/09/10 21:22:08] ppocr INFO:     dataset : 
[2023/09/10 21:22:08] ppocr INFO:         data_dir : /media/wj/newvolume/REC_DATASET/RCTW_rec/RCTW_rectrain_imgs/
[2023/09/10 21:22:08] ppocr INFO:         ext_op_transform_idx : 1
[2023/09/10 21:22:08] ppocr INFO:         label_file_list : ['/media/wj/newvolume/REC_DATASET/RCTW_rec/rec_train.txt']
[2023/09/10 21:22:08] ppocr INFO:         name : SimpleDataSet
[2023/09/10 21:22:08] ppocr INFO:         transforms : 
[2023/09/10 21:22:08] ppocr INFO:             DecodeImage : 
[2023/09/10 21:22:08] ppocr INFO:                 channel_first : False
[2023/09/10 21:22:08] ppocr INFO:                 img_mode : BGR
[2023/09/10 21:22:08] ppocr INFO:             RecConAug : 
[2023/09/10 21:22:08] ppocr INFO:                 ext_data_num : 2
[2023/09/10 21:22:08] ppocr INFO:                 image_shape : [32, 320, 3]
[2023/09/10 21:22:08] ppocr INFO:                 prob : 0.5
[2023/09/10 21:22:08] ppocr INFO:             RecAug : None
[2023/09/10 21:22:08] ppocr INFO:             CTCLabelEncode : None
[2023/09/10 21:22:08] ppocr INFO:             SVTRRecResizeImg : 
[2023/09/10 21:22:08] ppocr INFO:                 image_shape : [3, 32, 320]
[2023/09/10 21:22:08] ppocr INFO:                 padding : True
[2023/09/10 21:22:08] ppocr INFO:             KeepKeys : 
[2023/09/10 21:22:08] ppocr INFO:                 keep_keys : ['image', 'label', 'length']
[2023/09/10 21:22:08] ppocr INFO:     loader : 
[2023/09/10 21:22:08] ppocr INFO:         batch_size_per_card : 128
[2023/09/10 21:22:08] ppocr INFO:         drop_last : True
[2023/09/10 21:22:08] ppocr INFO:         num_workers : 8
[2023/09/10 21:22:08] ppocr INFO:         shuffle : True
[2023/09/10 21:22:08] ppocr INFO: profiler_options : None
[2023/09/10 21:22:08] ppocr INFO: train with paddle 2.3.2 and device Place(gpu:0)
[2023/09/10 21:22:08] ppocr INFO: Initialize indexs of datasets:['/media/wj/newvolume/REC_DATASET/RCTW_rec/rec_train.txt']
[2023/09/10 21:22:08] ppocr INFO: Initialize indexs of datasets:['/media/wj/newvolume/REC_DATASET/RCTW_rec/rec_val.txt']
W0910 21:22:08.206351 3318983 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.0, Runtime API Version: 10.2
W0910 21:22:08.207228 3318983 gpu_resources.cc:91] device: 0, cuDNN Version: 8.3.
[2023/09/10 21:22:08] ppocr INFO: train dataloader has 342 iters
[2023/09/10 21:22:08] ppocr INFO: valid dataloader has 49 iters
[2023/09/10 21:22:08] ppocr INFO: train from scratch
[2023/09/10 21:22:08] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by juneana123** [comment]
可以正常训练检测模型，但是在训练识别模型时GPU显示调用，但是只有1%,一直无法进行下一步的训练step

**Comment by hezhenname** [comment]
> 可以正常训练检测模型，但是在训练识别模型时GPU显示调用，但是只有1%,一直无法进行下一步的训练step

是不是你数据量太大了，这一步是要等很久，训练到第一步就快很多了

**Comment by juneana123** [comment]
我重新更换gpu版本后，显示调用了GPU。对于A数据集可以正常训练，但是对于B数据训练缓慢，而且A数据集数据量是大于B数据集的，这是怎么回事呢。
